cardano-explorer-core/test/test-db.hs

```diff
@@ -0,0 +1,19 @@
+
+import           Explorer.Core (LogFileDir (..), MigrationDir (..), PGPassFile (..), runMigrations)
+
+import           Test.Tasty (TestTree, defaultMain, testGroup)
+import           Test.Tasty.HUnit (testCase)
+
+main :: IO ()
+main =
+  defaultMain tests
+
+tests :: TestTree
+tests =
+  testGroup "Database"
+    [ testCase "Migration" testMigration
+    ]
+
+testMigration :: IO ()
+testMigration =
+  runMigrations True (PGPassFile "../config/pgpass") (MigrationDir "../schema") (LogFileDir "..")
```

Comment:
Is `config/pgpass` something that should exist in the repo, or does one have to make one to run the test?

Reply:
Yeah, need to figure that out.

Reply:
It has been figured out. Subsequent commits on this branch add that file (without any credentials).